packages/runner-docker/scripts/build.mjs

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env node
+
+import { spawn, exec } from 'child_process';
+import { promisify } from 'util';
+import process from 'process';
+
+const execAsync = async (cmd, opts) => {
+  console.log(`Executing: ${cmd}`);
+  return promisify(exec)(cmd, opts);
+};
+
+/**
+ * Check if Docker daemon is running
+ */
+async function isDockerRunning() {
+  try {
+    await execAsync('docker stats --no-stream', { timeout: 5000 });
+    return true;
+  } catch (error) {
+    return false;
+  }
+}
+
+/**
+ * Start Docker daemon (platform-specific)
+ */
+async function startDocker() {
+  console.log('Docker is not running. Attempting to start...');
+  
+  const platform = process.platform;
+  
+  if (platform === 'darwin') {
+    // macOS
+    try {
+      await execAsync('open /Applications/Docker.app');
+      console.log('Started Docker Desktop on macOS. Please wait for it to initialize...');
+    } catch (error) {
+      console.log('Could not auto-start Docker on macOS. Please start Docker Desktop manually.');
+    }
+  } else if (platform === 'win32') {
+    // Windows
+    try {
+      await execAsync('start "" "C:\\Program Files\\Docker\\Docker\\Docker Desktop.exe"');
+      console.log('Started Docker Desktop on Windows. Please wait for it to initialize...');
+    } catch (error) {
+      console.log('Could not auto-start Docker on Windows. Please start Docker Desktop manually.');
+      console.log('You can start Docker Desktop from the Start menu or by running:');
+      console.log('  "C:\\Program Files\\Docker\\Docker\\Docker Desktop.exe"');
+    }
+  } else {
+    // Linux - usually runs as a service
+    try {
+      await execAsync('sudo systemctl start docker');
```

Comment:
We should not ever call `sudo`. I get what you are doing here, but if the script does not have the ability to start docker without sudo then we should not start it ourselves.